lineman/app/components/dashboard_page/dashboard_page_controller.coffee

```diff
@@ -34,14 +34,14 @@ angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, $
     @currentBaseQuery = ThreadQueryService.filterQuery(@filter)
     if @displayByGroup()
       _.each @groups(), (group) =>
-        @views.groups[group.key] = ThreadQueryService.groupQuery(group, { filter: @filter, queryType: 'all' })
+        @views.groups[group.key] = ThreadQueryService.groupQuery(group, { filter: 'show_muted', queryType: 'all' })
```

Comment:
Oh! @filter is nothing more than a variable on the scope which is set by changing the 'currently_viewing' dropdown (recent, participating, muted, etc.)

So @filter should be set to 'show_muted', if the dropdown is set to that.